fc/auth/combo.py

```diff
@@ -12,15 +13,18 @@ def username(self):
             return self._username
 
         try:
-            return FileAuth().username()
+            return EnvVarAuth().username()
         except Exception:
-            return KeyboardAuth().username()
+            try:
```

Comment:
These nested `try` blocks should be OK for now.

Reply:
Yeah I didn't know if an`if` block before the `try` block was any better.

Reply:
I guess I can just rely on default `__init__` altogether then correct?

Reply:
I guess I can just rely on default `__init__` altogether then correct?